src/vmm/src/cpu_config/x86_64/static_cpu_templates/t2s.rs

```diff
@@ -259,7 +259,7 @@ pub fn t2s() -> CustomCpuTemplate {
             addr: 0x10a,
             bitmap: RegisterValueFilter {
                 filter: 0b1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111,
-                value: 0b0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_1100_0100_1100,
```

Comment:
Should we keep a comment like in the other commit of what the meaning of each bit is?

Reply:
Sorry, what do you mean? "Bit 19: RRSBA" has already been commented.

Reply:
ah sorry, I didn't expand the diff. I see it now